README.md

```diff
@@ -12,7 +12,7 @@
   <a href="https://github.com/KaotoIO/vscode-kaoto/actions/workflows/ci.yaml"><img src="https://img.shields.io/github/actions/workflow/status/KaotoIO/vscode-kaoto/ci.yaml?style=for-the-badge&label=released%20kaoto%20ui" alt="Build with Released version of Kaoto UI"></a>
 </p><br/>
 
-<h2 align="center">No Code and Low Code Integration editor</h2>
+<h2 align="center">Kaoto - The Open Source Visual Editor for Apache Camel</h2>
```

Comment:
not sure if `h2` need to contains also the `Kaoto - ` part because we have a big Kaoto logo on the top, but I don't not strong opinion here...